README.md

```diff
@@ -60,6 +60,7 @@ type XhrOptions = String | {
     withCredentials: Boolean?,
     responseType: String?,
     beforeSend: Function?
+    qs: Any?
```

Comment:
keep consistent w/ readme and avoid confusion of a non-type, i.e. `qs: Function?` here

> To support the `qs` option, you must define an `xhr.queryStringStringify` function. This function accepts the value of `options.qs` as its first argument, and returns a string that is appended to the URL.